scss/titles.scss

```diff
@@ -0,0 +1,24 @@
+$title-decoration-height: 90px;
+$title-decoration-width: 100px;
+
+.title {
+  font-size: $text-big;
+  margin: 100px 50px 50px 200px;  
+  position: relative;
+  
+  &::before {
+    background: url('assets/triangle.svg') no-repeat;
+    content: '';
+    height: $title-decoration-height;
+    left: -50px;
+    position: absolute;
+    top: -35px;
+    width: $title-decoration-width;
+    z-index: -100;
+  }
+}
+
+div {
```

Comment:
Don't style HTML tags directly! :scream:

Reply:
Keep in mind this is dangerous, specially if you need `div` tags that are either just wrappers or you need other kind of display